app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+    def self.search_food(search)
+        data = FoodService.search_food(search)
+        data[:foods].map do |food_data|
+            FoodInfo.new(food_data)
+        end
+    end
+
+    def self.search_count(search)
+        data = FoodService.search_food(search)
+        data[:totalHits]
+    end
+end
```

Comment:
In a follow up from my last comment, you might consider just having the `search_food` method, and letting that method return both the collection of food objects as well as the total hits.